drop-in/src/main/java/com/adyen/checkout/dropin/ComponentParsingProvider.kt

```diff
@@ -419,7 +421,7 @@ internal fun getComponentFor(
         CardComponent.PROVIDER.isPaymentMethodSupported(paymentMethod) -> {
             val cardConfig: CardConfiguration =
                 getConfigurationForPaymentMethod(paymentMethod, dropInConfiguration)
-            CardComponentProvider(dropInParams).get(
+            CardComponentProvider(dropInParams, checkoutSession).get(
```

Comment:
This should also be done in the stored component

Reply:
why do we need to do this? I mean stored component doesn't have that field.

Reply:
For now there is nothing from the configuration that is required for the stored component, but if we add later it'll be a simple change